### PR TITLE
[release/3.1] Documentation fixes

### PR DESCRIPTION
**Pull request by ljacomet**
This is a summary PR for all the changes proposed yesterday by @ChrisBradley001 
Let's review here and once this is ok, I will open the matching PR for master.

### CODE REVIEW

File: docs/src/docs/asciidoc/user/107.adoc [hunk 1]
```diff
@@ -11,15 +11,15 @@ endif::notBuildingForSite[]
...
+The JCache (aka JSR-107) specification defines the standard caching API for Java.
 The specification was developed under the Java Community Process v2.9 by an expert group including members from the Ehcache developer community.
 JCache provides a very simple API set that is easy to use and vendor neutral.
 
 Being one of the pioneers in the Java caching domain, Ehcache had to offer an implementation that is fully compliant with the JCache specification.
 
-For years the biggest problems that application developers have faced while wanting to try cache implementations by different vendors is the stark contrast in the APIs offered by these vendors.
-Developers were forced to rewrite a whole lot of their caching related code in an application just to try out a new caching solution.
-This lead to developers sticking with what they had as the bar to investigating other products was too high.
+For years, the biggest problems that application developers have faced while wanting to try cache implementations by different vendors is the stark contrast in the APIs offered by these vendors.
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/107.adoc]: biggest problem - that is drop the s?

File: docs/src/docs/asciidoc/user/clustered-cache.adoc [hunk 2]
```diff
@@ -34,31 +34,31 @@ Caches can reserve a storage area for their clustered tiers within these server
 Clustered Tier Manager::
 The Ehcache _Clustered Tier Manager_ is the server-side component that gives clustering capabilities to a cache manager.
 Cache managers connect to these to get access to the server's storage resources so that the clustered tiers of caches defined in them can consume those resources.
-Ehcache _Clustered Tier Manager_ at the server side are identified by a unique identifier.
+Each Ehcache _Clustered Tier Manager_ at the server side is identified by a unique identifier.
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/clustered-cache.adoc]: is uniquely identified.

File: docs/src/docs/asciidoc/user/clustered-cache.adoc [hunk 3]
```diff
@@ -34,31 +34,31 @@ Caches can reserve a storage area for their clustered tiers within these server
...
 Using it, multiple cache managers can connect to the same clustered tier manager to share cache data.
 The clustered tier manager is also responsible for managing the storage of the clustered tier of caches, with the following different options.
 
 Dedicated pool::
-Dedicated pools are a fixed-amount of storage pools allocated to the clustered tiers of caches.
+Dedicated pools are a fixed number of storage pools allocated to the clustered tiers of caches.
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/clustered-cache.adoc]: Dedicated pools represent a fixed quantity of storage allocated to the clustered tier of a cache.

File: docs/src/docs/asciidoc/user/clustered-cache.adoc [hunk 4]
```diff
@@ -34,31 +34,31 @@ Caches can reserve a storage area for their clustered tiers within these server
...
 A dedicated amount of storage is allocated directly from server off-heap resources to these pools.
 And this storage space is used exclusively by a given clustered tier.
 
 Shared pool::
-Shared pools are also fixed-amount storage pools, but can be shared by the clustered tiers of multiple caches.
+Shared pools are also a fixed number of storage pools, but can be shared by the clustered tiers of multiple caches.
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/clustered-cache.adoc]: Shared pools represent a quantity of storage to be share by the clustered tiers of multiple caches.

File: docs/src/docs/asciidoc/user/107.adoc [hunk 5]
```diff
@@ -50,23 +50,23 @@ include::{sourcedir31}/107/src/test/java/org/ehcache/docs/EhCache107Configuratio
...
 <9> Retrieve the data from the same cache.
 
 == JSR-107 and Ehcache configuration integration
 
-As mentioned already the JCache specification offers a minimal set of configuration that is ideal for an in-memory cache.
-But Ehcache native APIs support topologies that are much more complex and provides more features.
-At times application developers might want to configure caches that are much complex (in terms of topology or features)
-than the ones that JCache `MutableConfiguration` permits and still be able to use the JCache's caching APIs.
-Ehcache provides several ways to achieve that and this section covers the same.
+As mentioned already, the JCache specification offers a minimal set of configuration that is ideal for an in-memory cache.
```
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/107.adoc]: Indeed

File: docs/src/docs/asciidoc/user/xml.adoc [hunk 6]
```diff
@@ -95,7 +95,7 @@ include::{sourcedir31}/xml/src/test/java/org/ehcache/docs/GettingStarted.java[ta
     to create your `CacheManager` instance using the `Configuration` from the `XmlConfiguration`
 
 We can also use `<cache-template>` declared in the XML file to seed instances of `CacheConfigurationBuilder`. In order
-to use a `<cache-template>` element from a XML file, e.g. the `/my-config.xml` contained this XML fragment:
+to use a `<cache-template>` element from a XML file, e.g. the `/my-config.xml` contains this XML fragment:
```
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/xml.adoc]: I would put "an XML file"

File: docs/src/docs/asciidoc/user/107.adoc (hunk 1 above)
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/107.adoc]: :-) yes, drop the "s"

File: docs/src/docs/asciidoc/user/clustered-cache.adoc (hunk 2 above)
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/clustered-cache.adoc]: :-) I suggest "... at the server side has a unique identifier"

File: docs/src/docs/asciidoc/user/107.adoc (hunk 5 above)
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/107.adoc]: :+1:

File: docs/src/docs/asciidoc/user/clustered-cache.adoc (hunk 3 above)
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/clustered-cache.adoc]: :+1:  This is OK, perhaps with a bit more detail than the comment from ljacomet above.

File: docs/src/docs/asciidoc/user/clustered-cache.adoc (hunk 4 above)
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/clustered-cache.adoc]: :+1: This is OK, providing a bit more detail than the comment from ljacomet above.

File: docs/src/docs/asciidoc/user/107.adoc [hunk 7]
```diff
@@ -11,15 +11,15 @@ endif::notBuildingForSite[]
...
 JCache provides a very simple API set that is easy to use and vendor neutral.
 
 Being one of the pioneers in the Java caching domain, Ehcache had to offer an implementation that is fully compliant with the JCache specification.
 
-For years the biggest problems that application developers have faced while wanting to try cache implementations by different vendors is the stark contrast in the APIs offered by these vendors.
-Developers were forced to rewrite a whole lot of their caching related code in an application just to try out a new caching solution.
-This lead to developers sticking with what they had as the bar to investigating other products was too high.
+For years, the biggest problems that application developers have faced while wanting to try cache implementations by different vendors is the stark contrast in the APIs offered by these vendors.
+Developers were forced to rewrite a lot of their caching related code in an application just to try out a new caching solution.
+This lead to developers sticking with what they had, as the bar to investigating other products was too high.
```
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/107.adoc]: :+1:

File: docs/src/docs/asciidoc/user/xml.adoc (hunk 6 above)
Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/xml.adoc]: :+1:

**Comment by ljacomet** [comment]
Updated the PR with the remarks and squashed all commits into one. Ready to merge unless someone adds more comments.

**Pull request merged**